src/python/ddapp/teleoppanel.py

```diff
@@ -12,7 +12,8 @@
 from ddapp import drcargs
 from ddapp import affordanceurdf
 from ddapp.roboturdf import HandFactory
-import ddapp.applogic as app
+from ddapp import lcmUtils
+import drc
```

Comment:
There's probably a merge conflict at these import statements.  You'll have to rebase onto master or merge master.  For the import, change "ddapp" to "director".

Also, if you could, I usually do "import drc as lcmdrc" and then later refer to lcmtypes as lcmdrc.robot_state_t